### ISSUE TITLE
Cannot create angular-nest workspace version 14.7.5

### ISSUE THREAD

**Issue by khuyennguyen7007** [nrwl/nx#11996]
## Current Behavior
Cannot create angular-nest workspace version 14.7.5 but 14.7.4

## Expected Behavior
Working as expected

## Steps to Reproduce
Create workspace with the command:
```
npx create-nx-workspace@latest --preset=angular-nest --packageManager=yarn --name=angular-nest --appName=web --style=s
css --nxCloud=false
```
### Failure Logs
```
Need to install the following packages:
  create-nx-workspace@latest
Ok to proceed? (y)

 >  NX   Nx is creating your v14.7.5 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "yarn install" several times. Please wait.

✔ Installing dependencies with yarn
✖ Creating your workspace in angular-nest

 >  NX   Nx failed to create a workspace.

   Exit code: 1
   Log file: /tmp/tmp-4666-c2suWQ8OFvqK/error.log
```
error.log
```
yarn run v1.22.15
$ /tmp/tmp-4666-c2suWQ8OFvqK/node_modules/.bin/nx new angular-nest --preset=angular-nest --packageManager=yarn --pm=yarn --name=angular-nest --appName=web --style=scss --no-nxCloud --no-allPrompts --no-a --no-skipGit --no-g '--commit.message=Initial commit' --/bin/sh=../.npm/_npx/505743838affa773/node_modules/.bin/create-nx-workspace --defaultBase=main --ci= --collection=@nrwl/workspace/generators.json --cli=angular --nxWorkspaceRoot=//home/nx/workspaces

>  NX  Generating @nrwl/workspace/generators.json:new

CREATE angular-nest/README.md
CREATE angular-nest/.editorconfig
CREATE angular-nest/.gitignore
CREATE angular-nest/.prettierignore
CREATE angular-nest/.vscode/extensions.json
CREATE angular-nest/angular.json
CREATE angular-nest/package.json
CREATE angular-nest/tools/generators/.gitkeep
CREATE angular-nest/tools/tsconfig.tools.json
CREATE angular-nest/tsconfig.base.json
CREATE angular-nest/nx.json
CREATE angular-nest/.prettierrc
CREATE angular-nest/decorate-angular-cli.js
CREATE angular-nest/apps/.gitkeep
CREATE angular-nest/libs/.gitkeep
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@2.3.2: The platform "linux" is incompatible with this module.
info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
$ node ./decorate-angular-cli.js

 >  NX   Angular CLI has been decorated to enable computation caching.

$ nx g @nrwl/workspace:preset --name=web --style=scss --linter=eslint --npmScope=angular-nest --preset=angular-nest --packageManager=yarn --cli=ng --interactive=false

>  NX  Generating @nrwl/workspace:preset


 >  NX   Cannot find module 'eslint'

   Require stack:
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/utils/dist/ts-eslint/CLIEngine.js
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/utils/dist/ts-eslint/index.js
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/utils/dist/eslint-utils/RuleTester.js
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/utils/dist/eslint-utils/index.js
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/utils/dist/index.js
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/type-utils/dist/isTypeReadonly.js
   - /home/nx/workspaces/angular-nest/node_modules/@typescript-eslint/type-utils/dist/index.js
   - /home/nx/workspaces/angular-nest/node_modules/@nrwl/angular/src/generators/utils/insert-ngmodule-import.js
   - /home/nx/workspaces/angular-nest/node_modules/@nrwl/angular/src/generators/utils/index.js
   - /home/nx/workspaces/angular-nest/node_modules/@nrwl/workspace/src/generators/preset/preset.js
   - /home/nx/workspaces/angular-nest/node_modules/nx/src/config/workspaces.js
   - /home/nx/workspaces/angular-nest/node_modules/nx/src/config/configuration.js
   - /home/nx/workspaces/angular-nest/node_modules/nx/src/command-line/generate.js
   - /home/nx/workspaces/angular-nest/node_modules/nx/src/command-line/nx-commands.js
   - /home/nx/workspaces/angular-nest/node_modules/nx/bin/init-local.js
   - /home/nx/workspaces/angular-nest/node_modules/nx/bin/nx.js
   Pass --verbose to see the stacktrace.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

 >  NX   Workspace creation failed, see above.

   Pass --verbose to see the stacktrace.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

warning @nrwl/angular > @nrwl/web > stylus > css > source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
warning @nrwl/angular > @nrwl/web > css-minimizer-webpack-plugin > cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
warning "@nrwl/angular > @typescript-eslint/type-utils@5.37.0" has unmet peer dependency "eslint@*".
warning "@nrwl/angular > jasmine-marbles@0.8.4" has unmet peer dependency "rxjs@^6.5.3".
warning "@nrwl/angular > ts-node@10.9.1" has unmet peer dependency "@types/node@*".
warning "@nrwl/angular > @typescript-eslint/type-utils > @typescript-eslint/utils@5.37.0" has unmet peer dependency "eslint@^6.0.0 || ^7.0.0 || ^8.0.0".
warning "@nrwl/angular > @typescript-eslint/type-utils > @typescript-eslint/utils > eslint-utils@3.0.0" has unmet peer dependency "eslint@>=5".
[NX] Angular devkit called `writeWorkspace`, this may have had unintended consequences in angular.json
[NX] Double check angular.json before proceeding
error Command failed with exit code 1.
error Command failed with exit code 1.
```
### Environment
```
 >  NX   Report complete - copy this into the issue template

   Node : 16.11.1
   OS   : linux x64
   yarn : 1.22.15

   nx : 14.7.5
   @nrwl/angular : 14.7.5
   @nrwl/cypress : 14.7.5
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.7.5
   @nrwl/eslint-plugin-nx : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 14.7.5
   @nrwl/js : 14.7.5
   @nrwl/linter : 14.7.5
   @nrwl/nest : 14.7.5
   @nrwl/next : Not Found
   @nrwl/node : 14.7.5
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.7.5
   @nrwl/web : 14.7.5
   @nrwl/workspace : 14.7.5
   typescript : 4.8.3
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
```

**Comment by gRoussac** [comment]
Hi, I do reproduce this. It seems only the workspace gets created.

installing in the workspace directory with 

```
npx nx g @nrwl/angular:application my-app
npx nx g @nrwl/nest:app my-app
```
however does the job and thus install  "eslint": "~8.15.0" and "@nrwl/eslint-plugin-nx": "14.7.5"

**Comment by etrock** [comment]
I have the same issue.

**Comment by antonbarada** [comment]
same issue here create-nx-workspace@14.7.5

**Comment by FrozenPandaz** [comment]
I see this fixed on `14.7.11`:

```
jason@pop-os ~/p/triage> npx create-nx-workspace@latest ng-nest-test
npx: installed 60 in 3.527s
✔ What to create in the new workspace · angular-nest
✔ Application name                    · app1
✔ Default stylesheet format           · css
✔ Enable distributed caching to make your CI faster · Yes

 >  NX   Nx is creating your v14.7.12 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

✔ Installing dependencies with npm
✔ Nx has successfully created the workspace: ng-nest-test.
✔ NxCloud has been set up successfully

 >  NX   Successfully initialized git.


 ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————


 >  NX   First time using Nx? Check out this interactive Nx tutorial.

   https://nx.dev/angular-tutorial/01-create-application
   
   Prefer watching videos? Check out this free Nx course on YouTube.
   https://www.youtube.com/watch?v=2mYLe9Kp9VM&list=PLakNactNC1dH38AfqmwabvOszDmKriGco


 >  NX   Distributed caching via Nx Cloud has been enabled

   In addition to the caching, Nx Cloud provides config-free distributed execution,
   UI for viewing complex runs and GitHub integration. Learn more at https://nx.app
   
   Your workspace is currently unclaimed. Run details from unclaimed workspaces can be viewed on cloud.nx.app by anyone
   with the link. Claim your workspace at the following link to restrict access.
```

Can you all try with the `latest` please?

**Comment by khuyennguyen7007** [comment]
Hi @FrozenPandaz,
Confirm that it's working properly with the latest version (14.7.13).
```
 npx create-nx-workspace@latest --preset=angular-nest --packageManager=yarn --name=angular-nest --appName=web --style=scss --nxCloud=false

 >  NX   Nx is creating your v14.7.13 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "yarn install" several times. Please wait.

✔ Installing dependencies with yarn
✔ Nx has successfully created the workspace: angular-nest.
Switched to a new branch 'main'

 >  NX   Successfully initialized git.


 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————


 >  NX   Nx CLI is not installed globally.

   This means that you might have to use "yarn nx" or "npx nx" to execute commands in the workspace.
   Run "yarn global add nx" or "npm install -g nx" to be able to execute command directly.


 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————


 >  NX   First time using Nx? Check out this interactive Nx tutorial.

   https://nx.dev/angular-tutorial/01-create-application

   Prefer watching videos? Check out this free Nx course on YouTube.
   https://www.youtube.com/watch?v=2mYLe9Kp9VM&list=PLakNactNC1dH38AfqmwabvOszDmKriGco
```
```
 >  NX   Report complete - copy this into the issue template

   Node : 16.11.1
   OS   : linux x64
   yarn : 1.22.15

   nx : 14.7.13
   @nrwl/angular : 14.7.13
   @nrwl/cypress : 14.7.13
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.7.13
   @nrwl/eslint-plugin-nx : 14.7.13
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 14.7.13
   @nrwl/js : 14.7.13
   @nrwl/linter : 14.7.13
   @nrwl/nest : 14.7.13
   @nrwl/next : Not Found
   @nrwl/node : 14.7.13
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.7.13
   @nrwl/web : Not Found
   @nrwl/workspace : 14.7.13
   typescript : 4.8.3
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
```
